### ISSUE TITLE
TypeError: slice indices must be integers or None or have an __index__ method

### ISSUE THREAD

**Issue by MangGuoBingqilin** [Illumina/SpliceAI#118]
Hi
I am a beginner. I would like to ask, -A parameter is I use my own gtf file to change to the format it needs, but in the run is, there is the following error. I am not able to solve it well. Can you give me some suggestions?
Thanks!

`Traceback (most recent call last):
  File "/home/GYW68/miniconda3/bin/spliceai", line 33, in <module>
    sys.exit(load_entry_point('spliceai==1.3.1', 'console_scripts', 'spliceai')())
  File "/home/GYW68/miniconda3/lib/python3.8/site-packages/spliceai-1.3.1-py3.8.egg/spliceai/__main__.py", line 72, in main
  File "/home/GYW68/miniconda3/lib/python3.8/site-packages/spliceai-1.3.1-py3.8.egg/spliceai/utils.py", line 148, in get_delta_scores
TypeError: slice indices must be integers or None or have an __index__ method`

**Comment by MangGuoBingqilin** [comment]
This is what line 148 of the utils.py reads:
`x_ref = 'N'*pad_size[0]+seq[pad_size[0]:wid-pad_size[1]]+'N'*pad_size[1]`

**Comment by kishorejaganathan** [comment]
Could you share what your custom annotation file looks like, just the first 3-4 lines is enough.

**Comment by MangGuoBingqilin** [comment]
OK. The following is my custom annotation file. Please help to see if there is any problem. Thank you!
![18351930fd3d8aad72c5f9b2da5caa2](https://user-images.githubusercontent.com/128884398/230523185-58fe1dd1-daf3-480f-bda8-96856858e7ed.png)

**Comment by kishorejaganathan** [comment]
- The first column with 1,2,3,4 etc might be causing some problems, esp. if it is not tab separated. You can try removing that.
- Are you using the parameter D? If yes, it should be integer.
- Are you able to annotate the examples/input.vcf file properly and get examples/output.vcf as the output?

**Comment by MangGuoBingqilin** [comment]
Hello. I just tried it and the vcf file works fine. I didn't use the parameter D. It can calculate part of the SNP and will not report an error until a certain SNP site. And I looked in the vcf file and it has no other problems compared to other SNPs except for the different location. I looked at the annotation file to see if it was possible that there were duplicate sites in the EXON_STRAT and EXON_END. I'm trying to remove duplicate sites.
